pkg/agents/agents.go

```diff
@@ -17,6 +19,14 @@ import (
 	_ "github.com/gptscript-ai/gptscript/pkg/loader/github"
 )
 
+var emptyMessagesLimit = 500
+
+func init() {
+	if limit, err := strconv.Atoi(os.Getenv("CLI_CLICKY_CHATS_EMPTY_MESSAGES_LIMIT")); err == nil {
```

Comment:
```suggestion
	if limit, err := strconv.Atoi(os.Getenv("CLICKY_CHATS_STREAM_EMPTY_MESSAGES_LIMIT")); err == nil {
```

Reply:
Code completion added the `CLI_`. Thanks for catching.